test/JDBC/upgrade/13_6/schedule

```diff
@@ -294,3 +294,4 @@ ISC-Table_Constraints
 sys_server_principals_dep_for_13_x
 sys_database_principals_dep_for_13_x
 case_insensitive_collation
+BABEL-3507-before-15-1
```

Comment:
1. Please use a more descriptive test name other than BABEL-3507.
2. It should have `before-15-2-or-14-7` suffix because our current dev cycle is targeting PG15.2 and PG14.7 (cc: @rishabhtanwar29)
3. `BABEL-3507-before-15-1` and `BABEL-3507` has only one line difference. In that case, please consider creating three test files to avoid duplicates, e.g., `<test name>-common`, `<test name>-type-before-15-2-or-14-7` and `<test name>-type`.

Reply:
Fixed in the latest commit,
Divide existing test into common and Dependency tests